test/python3/libdnf/rpm/test_reldep_list.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (C) 2020 Red Hat, Inc.
+#
+# This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+#
+# Libdnf is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# Libdnf is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+import os
+
+import libdnf
+
+
+class TestReldepList(unittest.TestCase):
+    def setUp(self):
+        self.base = libdnf.base.Base()
+
+        # Sets path to cache directory.
+        cwd = os.getcwd()
+        self.base.get_config().cachedir().set(libdnf.conf.Option.Priority_RUNTIME, cwd)
+
+        self.repo_sack = libdnf.rpm.RepoSack(self.base)
+        self.sack = libdnf.rpm.SolvSack(self.base)
+
+        # Creates new repositories in the repo_sack
+        repo = self.repo_sack.new_repo("dnf-ci-fedora")
+
+        # Tunes repositotory configuration (baseurl is mandatory)
+        repo_path = os.path.join(cwd, "../../../test/libdnf/rpm/repos-data/dnf-ci-fedora/")
+        baseurl = "file://" + repo_path
+        repo_cfg = repo.get_config()
+        repo_cfg.baseurl().set(libdnf.conf.Option.Priority_RUNTIME, baseurl)
+
+        # Loads repository into rpm::Repo.
+        repo.load()
+
+        # Loads rpm::Repo into rpm::SolvSack
+        clsSolvSack = libdnf.rpm.SolvSack
+        self.sack.load_repo(repo.get(), libdnf.rpm.SolvSack.LoadRepoFlags_USE_FILELISTS |
+                            libdnf.rpm.SolvSack.LoadRepoFlags_USE_OTHER |
+                            libdnf.rpm.SolvSack.LoadRepoFlags_USE_PRESTO |
+                            libdnf.rpm.SolvSack.LoadRepoFlags_USE_UPDATEINFO)
+
+    def test_add(self):
+        a = libdnf.rpm.Reldep(self.sack, "python3-labirinto = 4.2.0")
+        b = libdnf.rpm.Reldep(self.sack, "(lab-list if labirinto.txt)")
+
+        list1 = libdnf.rpm.ReldepList(self.sack)
+        self.assertEqual(list1.size(), 0)
+        list1.add(a)
+        list1.add(b.get_id())
```

Comment:
This is not a problem, but a question to think about. Shall we have add() with ReldepId as a part of public API?

Reply:
I added TODO.